pom.xml

```diff
@@ -585,12 +585,27 @@
       </dependency>
       <dependency>
         <groupId>org.elasticsearch.plugin</groupId>
+        <artifactId>ingest-attachment</artifactId>
+        <version>${org.elasticsearch.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.elasticsearch.plugin</groupId>
         <artifactId>mapper-attachments</artifactId>
         <version>${org.elasticsearch.version}</version>
       </dependency>
       <dependency>
+        <groupId>org.codelibs.elasticsearch.module</groupId>
```

Comment:
We don't need to add it in our local repository since it's an artifact provided by ES:
https://mvnrepository.com/artifact/org.codelibs.elasticsearch.module/ingest-common